### PR TITLE
Add MAINTAINERS and Contributing.md

### PR DESCRIPTION
**Pull request by thaJeztah**
ping @stevvooe I just picked the top-3 contributors, but let me know if these should all be in the maintainers file. Added myself as "docs maintainer", although I don't think there's much docs-only things (but I can help getting small typo fixes etc LGTM'd faster)

### CODE REVIEW

File: MAINTAINERS [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+# To extract its contents programmatically, use any TOML-compliant parser.
+#
+# This file is compiled into the MAINTAINERS file in docker/opensource.
+#
+[Org]
+	[Org."Core maintainers"]
+		people = [
+			"aduermael",
+			"djs55",
+			"stevvooe",
```
Comment (stevvooe) [inline MAINTAINERS]: Let's add @FrenchBen and @justincormack.

**Comment by stevvooe** [comment]
@thaJeztah Looking good. I added a few more on there.

**Comment by thaJeztah** [comment]
@stevvooe cool, I'll update when I get back to my computer

**Comment by thaJeztah** [comment]
@stevvooe @FrenchBen @justincormack updated, PTAL

**Comment by stevvooe** [comment]
LGTM

**Pull request merged**